app/src/main/java/org/simple/clinic/drugs/PrescribedDrug.kt

```diff
@@ -134,5 +134,8 @@ data class PrescribedDrug(
         instantToCompare: Instant,
         pendingStatus: SyncStatus
     ): Boolean
+
+    @Query("SELECT uuid FROM PrescribedDrug WHERE patientUuid = :patientUuid")
+    fun prescribedDrugsUuidForPatient(patientUuid: UUID): List<UUID>
```

Comment:
We should be excluding deleted prescribed drugs from this list, otherwise it'll fetch every single prescribed drug from the beginning of time.

Reply:
Also, not sure if you're aware, but prescribed drugs have an `isDeleted` flag which we use instead of the `deletedAt` property since deleting prescribed drugs predates the `deletedAt` addition.

Reply:
Wait, I was not aware of that. Why does `deletedAt` exist then?

Reply:
When we first started, only prescribed drugs had a "delete" feature in the app and we implemented it using the `isDeleted` property. Later on, we realised that all the resources needed a way for us to mark them as deleted and we added the `deletedAt` method to all resources including the prescribed drugs.

But prescribed drugs can't be moved that easily since a lot of code on the server side and reports depends on the `isDeleted` flag so we didn't migrate it to use the `deletedAt` property yet since it requires some coordinated effort on both the backend and frontend to support.